Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp

```diff
@@ -89,6 +89,32 @@ static int connectionOptions()
     return IPC::PlatformConnectionOptions::SetCloexecOnClient | IPC::PlatformConnectionOptions::SetCloexecOnServer;
 }
 
+static bool isSandboxEnabled(const ProcessLauncher::LaunchOptions& launchOptions)
+{
+#if !USE(SYSTEM_MALLOC)
+    if (RUNNING_ON_VALGRIND)
+        return false;
+#endif
+
+    if (auto* sandboxEnv = g_getenv("WEBKIT_DISABLE_SANDBOX_THIS_IS_DANGEROUS")) {
```

Comment:
const auto*

Reply:
Hm, OK. I'll change them to `const char*` since there's not any benefit to writing `auto` instead of `char`.